src/js/hca/utils/validations.js

```diff
@@ -316,6 +316,15 @@ function isValidServiceInformation(data) {
 }
 
 function isValidForm(data) {
+  let isValidFinancialDisclosureSections = true;
```

Comment:
I was hitting a validation error when testing the form locally. I'm not sure if this has been failing all along or not. I had entered that I was married, but declined to provide financial info. Where this failed was when trying to validate the spouse's full name. Because the name was empty (since I skipped these fields), it failed validation when running `isValidForm` --> `isValidSpouseInformation` on submit. I added the check first in `isValidForm` that it should only validate those sections that conditionally appear in the financial disclosure section if the value of `discloseFinancialInformation` is true.